docs/src/pages.ts

```diff
@@ -1,67 +1,31 @@
 import type { MuiPage, OrderedMuiPage } from 'docs/src/MuiPage';
-import standardNavIcons from 'docs/src/modules/components/AppNavIcons';
 
 const pages: readonly MuiPage[] = [
   { pathname: '/versions' },
+  { pathname: '/docs/callouts', title: 'Callouts' },
   {
-    pathname: 'https://mui.com/store/',
-    title: 'Templates',
-    icon: standardNavIcons.ReaderIcon,
-    linkProps: {
-      'data-ga-event-category': 'store',
-      'data-ga-event-action': 'click',
-      'data-ga-event-label': 'sidenav',
-    },
-  },
-  { pathname: '/blog', title: 'Blog', icon: standardNavIcons.BookIcon },
-  {
-    pathname: '/x/react-data-grid-group',
-    title: 'Data Grid',
+    pathname: '/about',
+    title: 'Main parent',
```

Comment:
I'm confused why this one is rendering with the additional left padding (32px)? 🤔 If it's a first-level parent, it should render like the screenshot on the right, right?

| Additional padding | No padding (correct) |
|--------|--------|
| <img width="250" alt="Screen Shot 2023-07-19 at 11 09 24" src="https://github.com/mui/material-ui/assets/67129314/bbea762d-2776-4a68-b752-85b26ff327bd"> | <img width="250" alt="Screen Shot 2023-07-19 at 11 09 35" src="https://github.com/mui/material-ui/assets/67129314/3faf2a68-907c-4683-8c68-999b9caccbcf"> |

Reply:
Because the "Getting started" page has an icon:

https://github.com/mui/material-ui/blob/48e600b388c400f95f9232ae8bb652dad4603e7b/docs/data/material/pages.ts#L7-L9

This overrides the left padding in

https://github.com/mui/material-ui/blob/48e600b388c400f95f9232ae8bb652dad4603e7b/docs/src/modules/components/AppNavDrawerItem.js#L93-L95

I believe `hasIcon` has to be removed completely, and this one has to be rewritten to decrease the padding of the first-level items 

https://github.com/mui/material-ui/blob/48e600b388c400f95f9232ae8bb652dad4603e7b/docs/src/modules/components/AppNavDrawerItem.js#L47